code/game/atoms.dm

```diff
@@ -675,3 +675,8 @@ Proc for attack log creation, because really why not
 		return FALSE
 
 	return TRUE
+
+
+// For special click interactions (take first item out of container, quick-climb, etc.)
+/atom/proc/specialclick(mob/living/carbon/user)
+	return
```

Comment:
```suggestion
	return

```
Always end the files with a blank line.

Reply:
Still an issue. Please don't resolve things on your own, leave it to the reviewers.